README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About Me
+
+Hello! My name is **Nikhil Mandalaparthy** (my first name is pronounced *nick-hill*). I'm currently a MPP student at the Harris School of Public Policy, and I just graduated in June from the College, with a BA in Public Policy Studies. This is my [personal website](https://nikhilmandala.com/).
+
+## What I Look Like
+
+![Me](Headshot - Cropped.jpg){ width=50% }
+
+## My Biography
+
+Here's some information about my life -- where I'm from and where I've been.
+
+- My parents are from **India** -- specifically, the state of **Andhra Pradesh**.
+- I was born in **Singapore**.
+- When I was less than a year old, we moved to the **US**: first **Boston** for a few months and then **Seattle**
+- I spent the next 18 years of my life in **Seattle**; my younger brother was born there too.
+- After high school, I came to **Chicago** for college, and after finishing my bachelor's degree at UChicago, I'm staying on for an extra year to finish my MPP.
+- I've been lucky enough to visit **India** several times with my family. I also studied abroad in **Morocco** in 2018. I've visited a few other countries: **Indonesia**, **Mexico**, **Canada**, and **France**.
+- This past summer, I was working in **Washington, DC**, and then spent a few weeks in **India**.
+
+## Working with R Markdown & Github
+
+I am familiar with R and R Markdown, having used them extensively in my statistics classes and Program Evaluation class last year. I also used R in my internship this past summer to analyze and visualize data. Thus, I feel comfortable with different types of formatting, inserting code, etc. I hadn't embedded images or hyperlinks in a Markdown document before, so I had to Google how to do that and quickly found an answer.
+
```

Comment:
Just a heads up: Professor Soltoff included an online syntax guide for Markdown in the homework instructions. It obviously wasn't a big deal this week, but it may be helpful to carefully review the links in the instructions in the future.

Reply:
Good to know that you have touched upon some knowledge about R in your previous courses at Harris, and Im sure this will help you in the upcoming assignments in this course.